draft-ietf-dprive-dnsoquic.md

```diff
@@ -550,30 +550,32 @@ to the following recommendation:
 
 ## Connection Handling
 
-"DNS Transport over TCP - Implementation Requirements" {{?RFC7766}} provides
+"DNS Transport over TCP - Implementation Requirements" {{!RFC7766}} provides
 updated guidance on DNS over TCP, some of which is applicable to DoQ. This
 section attempts to specify which and how those considerations apply to DoQ.
 
 ### Connection Reuse
 
 Historic implementations of DNS clients are known to open and close TCP
-connections for each DNS query. To avoid excess QUIC connections, each with a
-single query, clients SHOULD reuse a single QUIC connection to the recursive
-resolver.
+connections for each DNS query. As outlined in section 6.2.1 of {{!RFC7766}}, to
```

Comment:
```suggestion
connections for each DNS query. As outlined in Section 6.2.1 of {{!RFC7766}}, to
```

I'd move the `{{` as well, but that can wait until Christian has updated his copy of kramdown.

Reply:
```suggestion
connections for each DNS query. To
```